sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/CBDE/MlirExportTest.cs

```diff
@@ -0,0 +1,1343 @@
+﻿extern alias csharp;
+using System.IO;
+using csharp::SonarAnalyzer.ControlFlowGraph.CSharp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SonarAnalyzer.UnitTest.CBDE
+{
+    [TestClass]
+    public class MlirExportTest
```

Comment:
if this is the test that just verifies the generated code to respect the MLIR syntax, please add a comment above the class to specify that